bin/plugin/commands/since.js

```diff
@@ -38,13 +43,32 @@ exports.handler = async ( opt ) => {
 		return;
 	}
 
-	const patterns = [
-		path.resolve( __dirname, '../../../**/*.php' ),
-		path.resolve( __dirname, '../../../**/*.js' ),
-	];
+	if ( opt.plugin && ! plugins.includes( opt.plugin ) ) {
```

Comment:
We also need to support someone passing `performance-lab` (which should result in the default behavior).

By making it the default, we could use:
```suggestion
	if ( opt.plugin !== 'performance-lab' && ! plugins.includes( opt.plugin ) ) {
```